crates/bevy_math/src/cubic_splines.rs

```diff
@@ -1230,4 +1336,33 @@ mod tests {
         let rational_accelerations: Vec<_> = rational_b_spline.iter_accelerations(10).collect();
         compare_vectors(cubic_accelerations, rational_accelerations, "acceleration");
     }
+
+    /// Test that a curbs curve can approximate a portion of a circle.
+    #[test]
+    fn nurbs_circular_arc() {
+        use std::f32::consts::SQRT_2;
+        const EPSILON: f32 = 0.1;
```

Comment:
This constant seems to be too big. Can it be reduced to something smaller? As it is, I think the allowed error would be very noticeable at such scale.